src/settings-ui/Settings.UI/SettingsXAML/Views/AdvancedPaste.xaml

```diff
@@ -76,7 +76,16 @@
                     </controls:SettingsGroup>
 
                     <controls:SettingsGroup x:Uid="AdvancedPaste_ClipboardHistorySettingsGroup" IsEnabled="{x:Bind ViewModel.IsEnabled, Mode=OneWay}">
-                        <tkcontrols:SettingsCard x:Uid="AdvancedPaste_Clipboard_History_Enabled_SettingsCard" HeaderIcon="{ui:FontIcon Glyph=&#xF0E3;}">
+                        <InfoBar
+                            x:Uid="GPO_SettingIsManaged"
+                            IsClosable="False"
+                            IsOpen="{x:Bind ViewModel.ClipboardHistoryDisabledByGPO, Mode=OneWay}"
+                            IsTabStop="{x:Bind ViewModel.ClipboardHistoryDisabledByGPO, Mode=OneWay}"
+                            Severity="Informational" />
+                        <tkcontrols:SettingsCard
+                            x:Uid="AdvancedPaste_Clipboard_History_Enabled_SettingsCard"
+                            HeaderIcon="{ui:FontIcon Glyph=&#xF0E3;}"
+                            IsEnabled="{x:Bind ViewModel.ClipboardHistoryDisabledByGPO, Mode=OneWay, Converter={StaticResource BoolNegationConverter}}">
                             <ToggleSwitch IsOn="{x:Bind ViewModel.ClipboardHistoryEnabled, Mode=TwoWay}" />
                         </tkcontrols:SettingsCard>
```

Comment:
plz switch order to be aligned with other settings. The bar is under the setting.

Reply:
@htcfreek  You mean I should switch the bar to be under the Setting? Currently it's above the Setting.

Reply:
I see. Let's make it below.